ocf_datapipes/transform/numpy/extend_timestamps_to_future.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
```

Comment:
why would we want to do this?

Reply:
PP needs the timestamps to go into the future for its forecasting, in training it deletes the future satellite data, but not the timestamps, but for production, and maybe to load less data in training, we can just extend the timestamps here, if that makes sense?